apps/i18n/codebridge/en_us.json

```diff
@@ -47,6 +47,7 @@
   "makeStarter": "Make starter file",
   "makeSupport": "Make support file",
   "makeValidation": "Make validation file",
+  "manageFiles": "Manage files",
```

Comment:
Nice name!
"fileOptions" was assigned "File Options". Do we want to use uppercase here as well?

Reply:
😅Updated!